src/javascript/binary/pages/client/form.js

```diff
@@ -114,7 +114,8 @@ ClientForm.prototype = {
                 var names = ['MAXCASHBAL', 'MAXOPENPOS',
                              'DAILYTURNOVERLIMIT', 'DAILYLOSSLIMIT',
                              '7DAYTURNOVERLIMIT', '7DAYLOSSLIMIT',
-                             'SESSIONDURATION', 'EXCLUDEUNTIL'];
+                             'SESSIONDURATION', 'EXCLUDEUNTIL',
+                             'REALITYCHECK'];
```

Comment:
remove. 1 hour for everyone. I'm really insistent on this. It's totally pointless and unnecessary to allow users to change the interval. Joanna needs to read the books Don't Make me Think and Getting Real.